lib/smoke_detector/providers/rollbar.rb

```diff
@@ -42,14 +42,13 @@ def default_client_settings
         </script>
       JS
```

Comment:
it should only be used in the precompile phase... or else we're doing something wrong